docs/features/test.md

```diff
@@ -0,0 +1,157 @@
+with default options.
+
+Additonally, 'scenarios' can be defined for each Feature to test more complicated cases (ie: Passing options to tests, testing >1 Feature together, etc.).  More information on [writing scenarios can be found below](#scenarios).
+
+The source code can of the sub-command is [here](../../src/spec-node/featuresCLI/test.ts). An example of the command being used in CI can be [found in the `devcontainers/features-template` repo](https://github.com/devcontainers/feature-template/blob/main/.github/workflows/test.yaml) and the [`devcontainers/features` repo](https://github.com/devcontainers/features).
+
+```
+devcontainer features test <target>
+
+Test features
+
+Positionals:
+  target  Path to folder containing 'src' and 'test' sub-folders.                     [string] [required] [default: "."]
+
+Options:
+      --help                   Show help                                                                       [boolean]
+      --version                Show version number                                                             [boolean]
+  -f, --features               Feature(s) to test as space-separated parameters. Omit to run all tests.  Cannot be
+                               combined with '--global-scenarios-only'.                                          [array]
+      --global-scenarios-only  Run only scenario tests under 'tests/_global' .  Cannot be combined with '-f'.
+                                                                                              [boolean] [default: false]
+      --skip-scenarios         Skip all 'scenario' style tests.  Cannot be combined with '--global--scenarios-only'.
+                                                                                              [boolean] [default: false]
+      --skip-autogenerated     Skip all 'autogenerated' style tests.                          [boolean] [default: false]
+  -i, --base-image             Base Image. Not used for scenarios.                            [string] [default: "ubuntu:focal"]
+  -u, --remote-user            Remote user. Not used for scenarios.                           [string] [default: "root"]
+      --log-level              Log level.                          [choices: "info", "debug", "trace"] [default: "info"]
+  -q, --quiet                  Quiets output                                                  [boolean] [default: false]
+```
+
+An example project structure can be found below.
+
+```
+.
+├── README.md
+├── src
+│   ├── dotnet
+│   │   ├── devcontainer-feature.json
+│   │   └── install.sh
+│   ├── oryx
+│   │   ├── devcontainer-feature.json
+│   │   └── install.sh
+|   ├── ...
+│   │   ├── devcontainer-feature.json
+│   │   └── install.sh
+├── test
+│   ├── _global
+│	│	├── scenarios.json
+│   │   └── some_test_scenario.json
```

Comment:
What is `some_test_scenario.json`? Can we have additional json files here? I don't see that mentioned anywhere in the text.

Reply:
A typo, thanks. It should be `some_test_scenario.sh`